Examples/Framework/include/ActsExamples/Utilities/tbbWrap.hpp

```diff
@@ -0,0 +1,169 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+// uncomment to remove all use of tbb library.
+// #define NO_TBB
+
+// #include <iostream>
+
+#ifdef NO_TBB
+#include <stdexcept>
+#else
+#include <tbb/parallel_for.h>
+#include <tbb/queuing_mutex.h>
+#include <tbb/task_arena.h>
+#endif
+
+namespace ActsExamples {
+
+/// Wrapper for most of the tbb functions that we use in Sequencer.
+/// It disables the use of tbb if nthreads=1.
+/// Note that only a small subset of tbb functions are implemented, and
+/// tbb::blocked_range (which doesn't require any thread setup) is still taken
+/// from the tbb library.
```

Comment:
This seems outdated, the current code reimplements `tbb::blocked_range`.

Reply:
it only reimplements it with `NO_TBB`, which I mention next. I should clarify a bit further.

Reply:
added to comment